tests/integration/tier1/destructive/changed-yum-conf/test_patch_yum_conf.py

```diff
@@ -9,6 +9,9 @@ def test_yum_conf_patch(convert2rhel, shell):
     expanding the $releasever variable properly.
     """
     shell("echo '#random text' >> /etc/yum.conf")
```

Comment:
This shell command should be placed little down, once we know which config we want to edit (either the dnf or yum conf)

Reply:
It doesn't need to since, `/etc/yum.conf` is either a config file or a symlink to `/etc/dnf/dnf.conf`
A comment would be nice though

Reply:
I see, thanks for the info. I haven't realized this. And +1 for the comment